ifc-js/src/App.vue

```diff
@@ -16,38 +15,31 @@ import {
   Scene,
   WebGLRenderer
 } from 'three'
-import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls'
-import { IFCLoader } from 'web-ifc-three/IFCLoader'
+import { OrbitControls } from 'three/addons/controls/OrbitControls.js'
+import { Components, IfcLoader } from '@thatopen/components'
```

Comment:
So, this is the replacement of the old web-ifc? Really nice that you manage to update to the latest version! Especially since the old had such a horrible "copy wasm" hack...